library/build.gradle.kts

```diff
@@ -1,5 +1,4 @@
 plugins {
-  id("java-library")
```

Comment:
Interesting, is adding this plugin optional?

Reply:
This should be handled by kotlin-jvm and plugin-publish.